pkg/collector/metric_collector.go

```diff
@@ -118,6 +123,7 @@ func (c *Collector) Update() {
 	// calculate the process energy consumption using its resource utilization and the node components energy consumption
 	if config.EnableProcessMetrics {
 		c.updateProcessEnergy()
+		c.updateVMEnergy()
```

Comment:
we'll evaluate the VM resolution overhead later and see if we need a `EnableVMMetrics` option

Reply:
I've took off all the calculations that are redundant with the Process metrics so the overhead should be very insignificant